test/helper.rb

```diff
@@ -1,4 +1,15 @@
+require 'vcr'
+
 if ENV['TRAVIS']
   require 'coveralls'
   Coveralls.wear!
 end
+
+VCR.configure do |config|
+  config.cassette_library_dir = 'test/cassettes'
+  config.hook_into :webmock
+  config.filter_sensitive_data('<AUTH_KEY>') { ENV['RENTLINX_AUTH_KEY'] }
+  config.filter_sensitive_data('<PASSWORD>') { ENV['RENTLINX_PASSWORD'] }
+  config.filter_sensitive_data('<USERNAME>') { ENV['RENTLINX_USERNAME'] }
```

Comment:
Could the file not live in the test directory? It's just for testing, there's no reason to push it out to who knows where.

We make a file like so:

    credentials:
      username: 'your_username_here'
      password: 'your_password_here'

in `test/credentials.yml` or something, commit it, then add it to the gitignore. All later changes to the file will be ignored by git until the file is specifically added with it's path (and then you have to acknowledge a warning). Documenting this is as easy as saying "add environment variables," but we're not forcing potential contributors to figure out how to do that on their OS.